crypto/src/signature.rs

```diff
@@ -227,10 +227,11 @@ impl<T: Encode> SignatureOf<T> {
 /// GUARANTEE 1: This container always contains at least 1 signature
 /// GUARANTEE 2: Each signature corresponds to a different public key
 #[allow(clippy::unsafe_derive_deserialize)]
-#[derive(Decode, Encode, Deserialize, Serialize, IntoSchema)]
```

Comment:
This wasn't correct because `SignaturesOf` invariants weren't checked on deserialization